src/test/java/com/example/rets_api/converter/PropertyConverterTest.java

```diff
@@ -126,4 +128,6 @@ private void checkAllFields_Property(PropertyEntity propertyEntity, PropertyDTO
         assertEquals(propertyEntity.getPropertyTypeTownHouse(), propertyDTO.getPropertyTypeTownHouse());
     }
 
+
```

Comment:
need to add LotData field at the tests dtoTOEntity_PropertyConverter_ShouldReturnCompleteObject and entityToDTO_RoomConverter_ShouldReturnCompleteObject